grid/chat/send-message/profile.supr

```diff
@@ -3,7 +3,7 @@ Send a message
 Send a text message to channel destination
 """
 name = "chat/send-message"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
This feels like different PR 🤔

Reply:
This is due to production reporting - it uses blocks for advanced formatting of message.